### PR TITLE
Improve documentation and argument parsing

### LINKED ISSUE #9: Remove header comments

### LINKED ISSUE #10: Improve documentation

### LINKED ISSUE #12: Fix bug when using `workon --new`

### PR DESCRIPTION
**Pull request by filipelbc**
Closes #9 
Closes #10 
Closes #12

### CODE REVIEW

File: dit.py [hunk 1]
```diff
@@ -2,41 +2,71 @@
...
+      Same as 'halt' followed by 'workon'
+
+    conclude [<id> | <name>]
+      Concludes the specified task or the current one. Note that there is an
+      implicit 'halt'
+
+    status [<id> | <name>]
+      Prints current task.
+
+    list [<id> | <name>] [--all, -a] [--concluded, -c]
```
Comment (filipelbc) [inline dit.py]: forgot `--verbose`

File: dit.py [hunk 2]
```diff
@@ -2,41 +2,71 @@
...
+      --all, -a
+        Lists all groups and subgroups.
+      --concluded, -a
+        Include concluded tasks in the listing.
 
     export [org]
+
+  <name>: ["group-name".]["subgroup-name".]"task-name"
+
+  <id>: --id, -i ["group-id".]["subgroup-id".]"task-id"
```
Comment (danielmoraes) [inline dit.py]: update "name" and "id" so that subgroup and task name are optional, group is mandatory

File: dit.py [hunk 3]
```diff
@@ -2,41 +2,71 @@
...
+      --new, -n
+        Same as 'new' followed by 'workon'.
+
+    halt [<id> | <name>]
+      Clocks out of the specified task or the current one.
+
+    switchto <id> | --new, -n <name> [-d "description"]
+      Same as 'halt' followed by 'workon'
+
+    conclude [<id> | <name>]
```
Comment (danielmoraes) [inline dit.py:41]: Conclude only makes sense for a task. Make it clear that one can not pass the ID of a group or a subgroup.

File: dit.py [hunk 4]
```diff
@@ -2,41 +2,71 @@
...
-    status
+    workon <id> | --new, -n <name> [-d "description"]
+      Clocks in the specified task.
+      --new, -n
+        Same as 'new' followed by 'workon'.
+
+    halt [<id> | <name>]
+      Clocks out of the specified task or the current one.
+
+    switchto <id> | --new, -n <name> [-d "description"]
```
Comment (danielmoraes) [inline dit.py:38]: switchto only makes sense for a task. Make it clear that one can not pass the ID of a group or a subgroup.

File: dit.py [hunk 5]
```diff
@@ -2,41 +2,71 @@
...
+    new <name> [-d "description"]
+      Creates a new task.
 
-    status
+    workon <id> | --new, -n <name> [-d "description"]
+      Clocks in the specified task.
+      --new, -n
+        Same as 'new' followed by 'workon'.
+
+    halt [<id> | <name>]
```
Comment (danielmoraes) [inline dit.py:35]: halt only makes sense for a task. Make it clear that one can not pass the ID of a group or a subgroup.

File: dit.py [hunk 6]
```diff
@@ -2,41 +2,71 @@
...
 
-    conclude ["id"]  [--group "id"] [--subgroup "id"]
+  <command>:
 
-    list [--group "id"] [--subgroup "id"] [--all]
+    new <name> [-d "description"]
+      Creates a new task.
 
-    status
+    workon <id> | --new, -n <name> [-d "description"]
```
Comment (danielmoraes) [inline dit.py:30]: workon only makes sense for a task. Make it clear that one can not pass the ID of a group or a subgroup.

**Pull request merged**